### PR TITLE
Clean qbittorrent package to respect convention

### PR DESCRIPTION
**Pull request by wget**
## Description
This package is often cited as an example of good package. However, the way installation and update process was made was outdated. The architecture bitness was still detected manually, while Chocolatey+AU could do it via the `file` and `file64` statements.

We removed a bunch of unneeded stuff and cleaned the package to respect convention (legal file path, no semi colon, etc.)

## Motivation and Context
Cleaning and avoid new comers to make issues when they check this package showcased as example.

## How Has this Been Tested?
Not tested. Waiting for CI build to complete.

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:
- [x] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md)
- [ ] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect a single package (not including meta package).

### CODE REVIEW

File: automatic/qbittorrent/tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -2,31 +2,17 @@
...
   silentArgs     = '/S'
   validExitCodes = @(0, 1223)
 }
 
 Install-ChocolateyInstallPackage @packageArgs
 
-# Lets remove the installer as there is no more need for it
-Remove-Item -Force "$toolsDir\*.exe","$toolsDir\*.exe.ignore" -ea 0
-
-$installLocation = Get-AppInstallLocation $packageArgs.softwareName
```
Comment (AdmiringWorm) [inline automatic/qbittorrent/tools/chocolateyinstall.ps1]: Please don't remove existing functionality from the package

File: automatic/qbittorrent/tools/chocolateyuninstall.ps1 [hunk 2]
```diff
@@ -1,26 +1,19 @@
...
-      packageName = $packageName
-      fileType    = 'exe'
-      silentArgs  = '/S'
-      validExitCodes= @(0)
-      file          = "$($_.UninstallString)"
-    }
-
-    Uninstall-ChocolateyPackage @packageArgs
-
-    $appPathKey = "HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\"
```
Comment (AdmiringWorm) [inline automatic/qbittorrent/tools/chocolateyuninstall.ps1]: same as the install script, please don't remove existing functionality

File: automatic/qbittorrent/tools/chocolateyuninstall.ps1 [hunk 3]
```diff
@@ -1,26 +1,19 @@
...
-    if (Test-Path "$appPathKey\qbittorrent.exe") { Remove-Item "$appPathKey\qbittorrent.exe" -Force }
-    if (Test-Path "$appPathKey\qbit.exe") { Remove-Item "$appPathKey\qbit.exe" -Force }
+  $packageArgs = @{
+    packageName = $packageName
+    fileType  = 'exe'
+    silentArgs  = '/S'
+    validExitCodes= @(0)
+    file      = "$($_.UninstallString)"
```
Comment (AdmiringWorm) [inline automatic/qbittorrent/tools/chocolateyuninstall.ps1]: `$_` is not available, as you've removed the iteration over the key array

File: automatic/qbittorrent/update.ps1 [hunk 4]
```diff
@@ -32,7 +32,7 @@ function global:au_GetLatest {
   $version64 = $url64 -split '[_]' | select -Last 1 -Skip 2
 
   if ($version -ne $version64) {
-    throw "32-bit and 64-bit version do not match. Please investigate."
+    throw "32-bit and 64-bit version do not match. Please contact the package manager."
```
Comment (AdmiringWorm) [inline automatic/qbittorrent/update.ps1]: You know this is a message to the package maintainers, right?
Basically now you're telling package maintainers to contact the package managers.
Please revert this line, as it doesn't really make sense.

File: automatic/qbittorrent/tools/chocolateyinstall.ps1 [hunk 5]
```diff
@@ -2,25 +2,20 @@
...
+  file64         = "$toolsDir\qbittorrent_4.1.1_x64_setup.exe"
   silentArgs     = '/S'
   validExitCodes = @(0, 1223)
 }
 
 Install-ChocolateyInstallPackage @packageArgs
 
 # Lets remove the installer as there is no more need for it
-Remove-Item -Force "$toolsDir\*.exe","$toolsDir\*.exe.ignore" -ea 0
+Get-ChildItem $toolsDir\*.exe | ForEach-Object { Remove-Item $_ -ea 0; if (Test-Path $_) { Set-Content "$_.ignore" '' } }
```
Comment (AdmiringWorm) [inline automatic/qbittorrent/tools/chocolateyinstall.ps1]: Just a tip, you don't need to use the full name of the cmdlets.
Aliases is automatically expanded when a package is updated.

Comment (wget) [reply automatic/qbittorrent/tools/chocolateyinstall.ps1]: Sure, but I find it more easier to read the code when the cmdlets are expanded. A personal matter of choice :)

Comment (AdmiringWorm) [reply automatic/qbittorrent/tools/chocolateyinstall.ps1]: fair enough.
Just wanted to let you know just in case.

Comment (AdmiringWorm) [reply automatic/qbittorrent/tools/chocolateyinstall.ps1]: also, just a little self promotion.
If you come across some scripts you want to expand aliases, and don't want to do it by hand.
Then you can download my powershell script through [`choco install wormies-au-helpers`](https://chocolatey.org/packages/wormies-au-helpers) (not really dependent on AU), where there is a helper script called [`Expand-Aliases`](https://wormiecorp.github.io/Wormies-AU-Helpers/docs/functions/expand-aliases) that you can use. (allows you to expand scripts in a directory, distinct files or just plain text).

**Pull request merged**